src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1221,7 +1223,16 @@ codet java_bytecode_convert_methodt::convert_instructions(
     else if(statement=="invokedynamic")
     {
       // not used in Java
-      code_typet &code_type=to_code_type(arg0.type());
+      code_typet &code_type = to_code_type(arg0.type());
+
+      const java_class_typet::java_lambda_method_handlet &lambda_method_handle =
+        lambda_method_handles.at(
+          std::stoi(id2string(code_type.get(ID_java_lambda_method_handle_id))));
+      const symbolt &lambda_method_symbol = symbol_table.lookup_ref
```

Comment:
So I can build directly on this - could this be moved into a method on/taking `lambda_method_handles` that gives me `java_lambda_method_handlet` and `lambda_method_symbol` for a given index (so for example I'm protected from changes in exactly how this works: `type().subtype().get(ID_identifier)`)